site/docs/v1/tech/migration-guide/firebase.adoc

```diff
@@ -0,0 +1,390 @@
+---
+layout: doc
+title: Migration From Firebase
+description: How to migrate your users to FusionAuth from Firebase.
+---
+:page-liquid:
+
+:migration_source_name: Firebase
+:migration_source_dir: firebase
+:script_supports_social_logins: false
```

Comment:
Why do we need the `/api/identity-provider/link` permission if we aren't supporting social logins?

Reply:
It looks like you support social logins: https://github.com/FusionAuth/fusionauth-import-scripts/pull/6/files#diff-492fa36046cc43dd051ae94842e3dcd3f5fb690ecb834e2d9d7a1ec5ef43954fR73

Reply:
yes, social logins are supported so we toggled this to true. Not really sure what it does though tbh.

Reply:
Thanks!

It's a switch consumed by this include file: site/docs/v1/tech/migration-guide/_social-login-note.adoc